packages/abi-coder/README.md

```diff
@@ -16,7 +16,9 @@ This module is responsible for encoding and decoding the Application Binary Inte
 
 ## Documentation
 
-See [Fuel-ts Documentation](https://fuellabs.github.io/fuels-ts/packages/fuel-ts-abi-coder/)
```

Comment:
These links used to link to the API reference page for the respective package in our old docs. We don't have API references in our new docs anymore, so I have replaced these links with a link to the home page of our docs.

I wouldn't even mind deleting these `Documentation` sections for packages that don't have docs as of yet. Thoughts?

Reply:
I agree with deleting them altogether, as we don't have a 1-1 relationship between the packages x doc's sections. Also, pointing them all to the main docs page sounds repetitive and unnecessary, IMO.

Reply:
Yes, we should delete those

Reply:
Thinking about this a second time, I think there might be some utility in keeping these links on all READMEs pointing to the main documentation page — repetitive as it may sound. Why? Because they will serve a purpose when browsing the packages via NPM.

@Dhaiwat10 @Torres-ssf What do you guys think?

 - https://www.npmjs.com/package/@fuel-ts/abi-coder

![doc](https://github.com/FuelLabs/fuels-ts/assets/26660/0a4d5df2-d3fa-4bc2-9e57-b074fec6c037)

Reply:
@arboleya, you raise an excellent point that we hadn't fully considered. Maintaining the links in all READMEs could indeed be beneficial for users browsing the packages via NPM.

Reply:
@arboleya that's a good point, but I'm not really sure if it would still be that useful because most of our users on npm will interact with the umbrella `fuels` package. I think it would be safe to assume that most people reading the `README` of any of these child packages would be browsing the source code on GitHub. This is why I see these readme files more as _internal_ files than external / user-facing.

Reply:
Yes, this one is a divider, but mostly I agree with your points.

I just wonder if having the link could still be a net-positive measure.

Also, although out of scope for this PR, the fact that the NPM packages don't link back to the GH repo is another issue. In the rare cases _(as you said)_ where users would go to the NPM page, it'd be a dead-end that doesn't connect anywhere (not the docs, not the repo). We should address this at some point.

Reply:
I also think there is some merit in keeping the link. Although I agree that most will be coming from `fuels`, the packages are independently indexed by the npm registry which often ranks highly on search engines. So potentially a rare case now but potentially more important in the future with more of our standardly named packages (`wallet`, `contract`, `address` etc.)

Reply:
Fair points, guys. I'm gonna add the links back since I do think it could be a net-positive.